projects/lights_control/inc/lights_events.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+typedef enum {
+  LIGHTS_EVENT_CAN_RX = 0, //
```

Comment:
Nit: you don't need the `//`s in places where you definitions are really long like this.

Reply:
what does Nit stand for?

Reply:
when my definitions are really long or when my definitions are not really long?